.github/workflows/config.yml

```
@@ -0,0 +1,51 @@
+name: config
+
+on:
+  push:
+    branches:
+      - master
+      - test
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

Let's use a newer version